spec/memfs/dir_spec.rb

```diff
@@ -216,6 +217,9 @@ module MemFs
       it_behaves_like 'returning matching filenames', '/test?', %w[/test0 /test1 /test2]
       it_behaves_like 'returning matching filenames', '/test[01]', %w[/test0 /test1]
       it_behaves_like 'returning matching filenames', '/test[^2]', %w[/test0 /test1]
+      it_behaves_like 'returning matching filenames', Pathname.new('/'), %w[/]
+      it_behaves_like 'returning matching filenames', Pathname.new('/*'),
+        %w[/tmp /test0 /test1 /test2]
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Align the parameters of a method call if they span more than one line.](https://www.codacy.com/app/happynoff/memfs/file/2685727877/issues/source?bid=3240728&fileBranchId=3336523#l222)